learning-bitmark/working-with-bitmarks/transferring-bitmarks.md

```diff
@@ -0,0 +1,358 @@
+    </div>
+
+    <br>
+
+* Observe changes on the recipient side after some minutes
+
+    <div style="background-color: #efefef; text-align: center;">
+        <img src="images/TransferBitmark_10.png" alt="Confirmed Properties screen" title="Confirmed Properties screen" width="250" style="padding: 20px" />
+        <img src="images/TransferBitmark_11.png" alt="Confirmed Details screen" title="Confirmed Details screen" width="250" style="padding: 20px" />
+    </div>
+
+<br>
+<br>
+
+## Transferring bitmarks using Bitmark SDK
+
+As a KIT for developers to interact with the Bitmark Property System, the Bitmark-SDKs provide interfaces for users to submit transactions to the Bitmark blockchain including the bitmark transfer.
+
+<br>
+
+Here are the steps to transfer bitmarks using the **Bitmark JS SDK**:
+
+* Submit a transfer transaction
+
+    ```js
+        let transferParams = Bitmark.newTransferParams(recipientAccount);
+        await transferParams.fromBitmark(bitmarkid);
+        transferParams.sign(sender);
+
+        let txs = (await Bitmark.transfer(transferParams)).txs;
+        let txId = txs[0].id;
+    ```
+
+<br>
+
+* Verify the transfer transaction
+
+    ```js
+        await Transaction.get(txId);
+    ```
+
+<br>
+<br>
+
+## Transferring bitmarks using Bitmark CLI
+
+The Bitmark CLI allows users to transfer bitmarks by submitting the transactions to its connected node, and then broadcasting to the network.
+
+<br>
+
+> The basic structure of a Bitmark CLI command:
+>   `bitmark-cli [global-options] command [command-options]`
+
+<br>
+
+> The Bitmark CLI determines which network to send the command by using the global option `--network` (abbreviation: `-n`). Option values:
+>
+>* `bitmark`:  the live network which uses live BTC or LTC to pay for the transactions.
+>
+>* `testing`:  a network for testing newly developed programs, it uses testnet coins to pay for transactions.
+>
+>* `local`: a special case for running a regression test network on the loopback interface.
+
+<br>
+
+Here are the steps to register a new Bitmark Certificate using the Bitmark CLI
+
+* Submit a transfer request
+
+    ```shell
+    $ bitmark-cli -n <network> -i <sender identity>\
+      transfer -r <receiver> \
+      -t <txid> -u
+    ```
+    > The `transfer` command is to submit a transfer transaction to the network. The options:
+    >
+    >* `receiver` - The identity of the recipient
+    >
+    > `txid` - The id of the last transaction of the bitmark which is transferring.
+    >
+    >**NOTE: ** A transfer transaction on the Bitmark blockchain requires a transaction fee of 0.0002 BTC or 0.002 LTC. Therefore, after the `transfer` command, the user needs to execute the payment for the transaction to be confirmed on the blockchain. The payment information is included in the output of the `transfer` command.
```

Comment:
Since the actual payments occur in satoshis or photons, you should mention those equivalents here.

Reply:
(e.g.: "0.0002 BTC (20000 satoshis)")

Reply:
Updated.